mmv1/third_party/terraform/.teamcity/components/builds/build_triggers.kt

```diff
@@ -36,7 +36,8 @@ class NightlyTriggerConfiguration(
 fun Triggers.runNightly(config: NightlyTriggerConfiguration) {
 
     schedule{
-        enabled = config.nightlyTestsEnabled
+        // enabled = config.nightlyTestsEnabled
+        enabled = false
```

Comment:
what's the reasoning for changing the default schedule trigger from true to false? Wouldn't we want to change this for the feature branch on these lines: https://github.com/GoogleCloudPlatform/magic-modules/pull/11847/files#diff-88a2ec67d77fa2a705a15f250a637d99cd7e6fd8f9ca524c2fddb25c9fa5ed3dR43-R45 ?

This would apply it to all projects disabling the nightly-test from being triggered, if my understanding is correct.

Reply:
The commit message explains- while the test deployment of this PR is up (link in the PR description) I've got the code disabling crons so we don't run nightly tests in duplicate.